Documentation/project-docs/developer-guide.md

```diff
@@ -165,26 +175,17 @@ We use the OSS testing framework [xunit](http://xunit.github.io/) with the [Buil
 #### Running tests on the command line
 
 By default, the core tests are run as part of `build.cmd` or `build.sh`. If the product binaries are already available, you could do `build-tests` which will build and run the tests.
-If the tests are already built and you only want to run them, invoke `run-tests`.
 
 For more information about cross-platform testing, please take a look [here](https://github.com/dotnet/corefx/blob/master/Documentation/building/cross-platform-testing.md).
 
 If you are interested in building and running the tests only for a specific library, then there are two different ways to do it:
 
-The easiest (and recommended) way to do it, is by simply building the test .builds file for that library.
-
-```cmd
-cd src\System.Collections.Immutable\tests
-msbuild System.Collections.Immutable.Tests.builds
-```
-
-What that will do is to build the tests against all of the available configurations, and then apply some filtering to run only the project configurations that support the default TestTFM (more about this in the next section). This is the preferred way since even though we only run the tests on a specific TFM, we will cross compile the tests against all its available configurations, which is good in order to make sure that you are not breaking the test build. This is also how CI will run the tests, so if you are able to build the .builds file successfully, chances are that CI will too.
-
-The second way to do it is to build the .csproj and select to run the `BuildAndTest` target:
+The easiest (and recommended) way to do it, is by simply building the test .csproj file for that library.
 
 ```cmd
 cd src\System.Collections.Immutable\tests
 msbuild /t:BuildAndTest (or /t:Test to just run the tests if the binaries are already built)
+msbuild /t:RebuildAndTest (this will cause a test project to rebuild and then run tests)
```

Comment:
Nit: I am curious if using proper cmd comments will do better syntax highlighting -> "& :: (this will ..."